### ISSUE TITLE
Please Push a Release Tag

### ISSUE THREAD

**Issue by edmondchui** [ilmente/mnTouch#9]
Hi,

Thanks for the great work. It really works on Android and iOS.

Could you please do a formal release tag? Currently I have to add the following line to my `bower.json` to make it work:

``` json
    "mnTouch": "https://github.com/ilmente/mnTouch.git#master",
```

Instead I want to do this

``` json
    "mnTouch": "1.2.2",
```

Thanks,
Edmond

**Comment by ilmente** [comment]
Hi Edmond,
sure I will! Give me a day. I'm in a busy situation.
Thanks for your feedback!

Best.
Ale

**Comment by ilmente** [comment]
Done!
https://github.com/ilmente/mnTouch/releases

Let me know if everything is fine!
Thanks!
Ale

**Comment by edmondchui** [comment]
Thanks for the quick action, @ilmente 

Unfortunately, I still get the following error

```
$ bower install
bower mnTouch#1.2.2          ENOTFOUND Package mnTouch not found
```

**Comment by ilmente** [comment]
I'll try some fix today.
Thanks!

**Comment by ilmente** [comment]
Hi @edmondchui,
sorry if I took a lot of time to do it, but now it works. Proof.

I created the new version `1.2.3` with a minor update regarding `bower`, so it works (I needed to publish the plugin in the public bower register in order to make it available with version numbers).
I added `npm` support as well. :)

```
$ bower install mn-touch 
$ bower install mn-touch#1.2.3
```

and

`$ npm install mn-touch`

Try and let me know!
Best.
